open-company/open-company-web
# Ms minor bugs

Issues:
- when i'm on MS and i remove the last MS i see MS disappear from the left navbar
- when i'm on MS and i remove a MSs it stays there w/o the blue bolt

To test:
- go to /carrot/must-see when you have no must sees
- [x] do you see Must See in the left nav bar highlighted? Good
- got to a board
- now add a Must See
- add another
- go to MS
- [x] do you see the correct Must See posts? Good
- remove both the posts from MS
- [x] do you see them disappear from MS when you remove them instead of only losing the blue blot? Good
